services/brig/src/Brig/API/Public.hs

```diff
@@ -1020,15 +997,15 @@ respFromActivationRespWithStatus = \case
 activateKeyH :: JSON ::: JsonRequest Public.Activate -> (Handler r) Response
 activateKeyH (_ ::: req) = do
   activationRequest <- parseJsonBody req
-  respFromActivationRespWithStatus <$> activate activationRequest
+  respFromActivationRespWithStatus <$> activate' activationRequest
 
-activateH :: Public.ActivationKey ::: Public.ActivationCode -> (Handler r) Response
-activateH (k ::: c) = do
+activate :: Public.ActivationKey -> Public.ActivationCode -> (Handler r) ActivationRespWithStatus
+activate k c = do
```

Comment:
(`functionNameH` traditionally means "`functionName`, but there is a servant handler with that name, and this is just the wrapper with the wai-route plumbing in it; will be removed once we switch to servant".  So it's probably/usually a good idea to merge the two, but yes, in this case I'm not sure how to do that off the top of my head.)